docs/pages/Hosting/Netlify.adoc

```diff
@@ -37,6 +37,9 @@ In order to follow the guide below without any problems, there are some precondi
 == Initial Setup Guide
 After this guide, you are the owner of your own PWA Template repository which builds are hosted in Netlify.
 If you are missing contextual information to any step, you might find it here https://docs.netlify.com/get-started/.
+To enhance the user experience of our Netlify documentation, we propose the addition of a "Deploy to Netlify" button. This button will enable users to swiftly deploy new websites from templates with a single click, streamlining the onboarding process for web developers and making it easier for them to deploy their applications on Netlify.
```

Comment:
I'd suggest to rewrite the second sentence 

"The following button will deploy a PWA template instance on Netlify, streamlining the onboarding process for web developers and making it easier for them to get started. You can use it and continue to the guide at chapter <<Configure and deploy Site>>